DeepCat.js

```diff
@@ -562,14 +565,32 @@
 	}
 
 	function addSmallFormHint() {
-		var smallHintBox = '<div id="deepcat-smallhint">'
+		var smallHintBox = '<div id="deepcat-smallhint" style="display: none;">'
 			+ '<img id="deepcat-smallhint-hide" title="' + mw.msg( 'deepcat-smallhint-close' ) + '" src="https://upload.wikimedia.org/wikipedia/commons/4/44/Curation_bar_icon_close.png">'
 			+ mw.msg( 'deepcat-hintbox-small' )
 			+ '</div>';
 		$( '#searchform' ).after( smallHintBox );
 		$( '#deepcat-smallhint-hide' ).on( 'click', hideSmallHint );
 	}
 
+	function initializeHintCodeIfNeeded() {
+		if( hintCodeLoaded ) {
+			return;
+		}
+		mw.loader.load( cssPath + 'DeepCat.hintbox.css&action=raw&ctype=text/css', 'text/css' );
+		addSearchFormHint();
+		addSmallFormHint();
+		hintCodeLoaded = true;
+	}
+
+	function loadThrobberCodeIfNeeded() {
```

Comment:
`code` seems a bit mysterious, then I check what it does, and it load styles. Maybe `loadThrobberStyles` or `loadThrobberResources`? I am not sure if I'd put the "IfNeeded" part to the method names but it might be just matter of taste, I am not saying you should change it one way or another.